docs/reference/release-notes/870.md

```diff
@@ -36,25 +36,38 @@ See the [release announcement](../announcements/870.md#amazon-web-services-aws-r
 
 ### BPMN Copilot BPMN-to-text
 
-Besides [generating BPMN](/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md), Camunda's BPMN Copilot now supports generating text from the BPMN diagram. This can be used to:
+<!--
+Camunda's BPMN Copilot now also supports generating text from a BPMN diagram.
+
+BPMN-to-text allows you to:
 
 - Rapidly draft process documentation.
 - Learn how the process works.
-- Explain the process's behavior to stakeholders.
+- Explain the behavior of the process to stakeholders.
 
 To learn more about this feature, see [BPMN Copilot](/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md).
 
-<!-- https://github.com/camunda/product-hub/issues/2576 -->
+https://github.com/camunda/product-hub/issues/2576
+-->
```

Comment:
This is commented out still. Is that intentional?

Reply:
It's still in QA, so not live yet: https://github.com/camunda/camunda-docs/pull/5184#issuecomment-2713666795